### PR TITLE
Delete moved ContainerSource from eventing-sources.

### PR DESCRIPTION
**Pull request by n3wscott**
Follow-up to https://github.com/knative/eventing/pull/1099

## Proposed Changes

  * Delete moved ContainerSource

**Release Note**

```release-note
ContainerSource now comes from installing eventing.
```

### CODE REVIEW

**Comment by n3wscott** [comment]
Samples will be moved as a follow-up.

**Comment by n3wscott** [comment]
Related to https://github.com/knative/docs/issues/1235

**Comment by vaikas-google** [comment]
Please also update Docs in knative/docs to reflect this. Those samples will also need to change.

**Comment by n3wscott** [comment]
> Please also update Docs in knative/docs to reflect this. Those samples will also need to change.

that work is being tracked here: https://github.com/knative/docs/issues/1235

**Comment by n3wscott** [comment]
error: 

```
(╯°□°)╯︵  kubectl  logs -n knative-sources controller-manager-0
2019/04/25 22:56:57 Registering Components.
2019/04/25 22:56:57 Skipping the AWS SQS Source controller.
2019/04/25 22:56:57 no kind is registered for the type v1alpha1.ContainerSource in scheme "k8s.io/client-go/kubernetes/scheme/register.go:60"
```

Will fix.

**Comment by n3wscott** [comment]
Fixed.

File: contrib/kafka/pkg/reconciler/kafkasource_test.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func init() {
 	v1.AddToScheme(scheme.Scheme)
 	corev1.AddToScheme(scheme.Scheme)
 	sourcesv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
-	genericv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
+	eventingsourcesv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
```
Comment (vaikas-google) [inline contrib/kafka/pkg/reconciler/kafkasource_test.go]: why not just use aws here and kafka in aws so we don't have to pull in eventingsources, or is there another reason for it?

Comment (n3wscott) [reply contrib/kafka/pkg/reconciler/kafkasource_test.go]: it needs ContainerSource from eventing

File: config/201-clusterrole.yaml [hunk 2]
```diff
@@ -34,12 +33,20 @@ rules:
   - patch
   - delete
 
+  # ContainerSource from eventing
```
[the remark this answers is not in the document]
Comment (n3wscott) [reply config/201-clusterrole.yaml]: Because there are sources that are built ontop of the ContainerSource, this brings ContainerSource into the schema

**Comment by n3wscott** [comment]
I wonder... I bet this needs to install the latest version of eventing, but I bet it is installing the latest release....

**Comment by n3wscott** [comment]
> but I bet it is installing the latest release....

it is installing the latest nightly release.

**Comment by n3wscott** [comment]
looks like it is back to the same error message.

/retest

**Comment by n3wscott** [comment]
I think the there is an issue with the upstream container source. Let me go fix...

**Comment by n3wscott** [comment]
Verified it works in eventing.

**Pull request merged**